service-api/module/Application/src/Controller/YotiController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Controller;
+
+use Application\Yoti\YotiServiceInterface;
+use Laminas\Mvc\Controller\AbstractActionController;
+use Laminas\Http\Response;
+use Laminas\View\Model\JsonModel;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ * Needed here due to false positive from Laminas’s uninitialised properties
+ */
+class YotiController extends AbstractActionController
+{
+    public function __construct(
+        private readonly YotiServiceInterface $yotiService
+    ) {
+    }
+
+    public function findPostOfficeAction(string $postCode): JsonModel
+    {
+        $branches = [];
+        $branches["locations"] = $this->yotiService->postOfficeBranch($postCode);
+        return new JsonModel($branches);
+    }
+    public function createSessionAction(array $sessionData): JsonModel
+    {
+        $data = [];
+        $data['response'] = $this->yotiService->createSession($sessionData);
+        return new JsonModel($data);
```

Comment:
I don't think we should return the data directly from Yoti's API, because then we end up with the front-end having a dependency on an external API. For now maybe worth just returning empty/204 until we have a need for any data to be passed back to the frontend.
```suggestion
        return new JsonModel(null);
```

We'll also need to save the session ID from the response to `CaseData` so that we can retrieve it again in the future.